CONTRIBUTING.md

```diff
@@ -136,6 +136,23 @@ npm run test:coverage
 
 See the [Create-React-App docs](https://github.com/facebook/create-react-app) for further information on JS testing.
 
+#### Testing package imports
+
+You can simulate how the published package will behave when imported into another JavaScript application by running
+
+```bash
+npm run lib-test
+```
+
+This script uses `npm pack` to package Kedro-Viz as a tarball, then copies it to a boilerplate React app in `/tools/test-lib/react-app` and installs it. Next, the script runs a simple server at `http://localhost:1337` so you can check that everything is working as it should.
```

Comment:
"...so you can check that everything is working as it should."

How would you do that? Does this mean it opens the page in the browser and you can see output? In which case, maybe clarify. Again, probably because I know nothing of this topic, but I think it may be helpful

Reply:
Yes, it opens the page in the browser automatically - it looks like this:
![image](https://user-images.githubusercontent.com/1155816/89540874-8960c000-d7f5-11ea-96e3-503c68b75e65.png)

I'd hope it might be reasonanly self-explanatory, but I can expand upon this point if you like?

Reply:
It's mostly an internal-use thing anyway, but I wanted to document it just in case someone finds it useful, and also so that we have documentation of it for new starters.

Reply:
I think it probably makes sense to make a slight edit, as follows, or something similar...no emphasis needed, just added for clarity.

"Next, the script runs a simple server at `http://localhost:1337`, **which opens in the browser,** so you can check that everything is working as it should."

??

Reply:
Makes sense, will do!